forge-gui-desktop/src/main/java/forge/screens/deckeditor/controllers/ACEditorBase.java

```diff
@@ -576,5 +580,24 @@ public void addMoveAlternateItems(final String verb, final String dest) {
                     InputEvent.SHIFT_DOWN_MASK | Toolkit.getDefaultToolkit().getMenuShortcutKeyMask(),
                     InputEvent.ALT_DOWN_MASK | Toolkit.getDefaultToolkit().getMenuShortcutKeyMask());
         }
+        public void addSetColorID() {
+            String label = localizer.getMessage("lblColorIdentity");
+            CardManager cardManager = (CardManager) CDeckEditorUI.SINGLETON_INSTANCE.getCurrentEditorController().getDeckManager();
+            PaperCard existingCard = cardManager.getSelectedItem();
+            int val;
+            if ((val = existingCard.getRules().getSetColorID()) > 0)
```

Comment:
wrong logic since it's supposed to be 2?

Reply:
The value is set by the card script setcolorid:val, and the editor wasnt for cryptic spires only. The setting for colorid can be reused.

Reply:
that's great and all but I'm telling you it just doesn't work right now...

Reply:
Seems to work here, are you on the catalog menu? You ahould be in the main or side board